Push data to google drive

alexwolson opened the pull request:
Almost completely contained within the `drive.py` file.

Quick rundown:

- Using this requires access to the torontobids@gmail.com account I made. I can share the required `token.pickle` file directly to anyone who wants to test this, I suppose. 
- The main method is `upload_directory`. This recursively creates folders and then uploads the files within them. It checks if these files and folders exist first, so it won't upload a file which is already there.
- `upload_file` does what it says - it also specifies the mime type of each file, for better preview on google drive.

This is tested & working on my end. Fairly straightforward implementation once I got the hang of all the oauth2 stuff.

sonalranjit commented on scrapers/ariba/drive.py:
Would be it better to name this `google_drive.py` or `google_drive_adapter.py` ?
something less generic so it'll be easier to understand from a overview look

sonalranjit commented on scrapers/ariba/drive.py:
```diff
@@ -0,0 +1,85 @@
+import pickle
+
+
+class Drive:
```
Same comment as above with the filename

sonalranjit reviewed (commented):
We should get into the habit or running the `Black` formatter when creating PRs, it'll be easier to read the code where it all comes in the same style/format 😬

alexwolson commented on scrapers/ariba/drive.py:
```diff
@@ -0,0 +1,85 @@
+# %%
+if __name__ == '__main__':
+    drive = Drive()
+    drive.recursively_upload_directory('data', drive.create_folder('data'))
```
I think I actually disagree. Part of the goal of this scraper is that it should upload without duplicates, using the same directory structure, regardless of where it's being run. So there should be no conflicts even if multiple people are uploading. It works this way now - files are uploaded based on the Doc ID, and it checks for the existence of files before uploading.

alexwolson commented:
Sonal, I addressed all of your comments in my latest commit except for the comment about a separate development folder. Thank you for your feedback! I'm learning a lot.

sonalranjit commented:
> Sonal, I addressed all of your comments in my latest commit except for the comment about a separate development folder. Thank you for your feedback! I'm learning a lot.

Reviewed the changes, looks awesome!
